README.md

```diff
@@ -38,3 +38,9 @@ Note that running the standalone binaries without root requires some extra capab
 ```sh
 sudo setcap 'cap_net_raw,cap_net_admin+eip' ./ac2mqtt
 ```
+
+### Troubleshooting
+
+**send ATT request failed: io: read/write on closed pipe, mac: xx:xx:xx:xx:xx:xx**
```

Comment:
is this still accurate if the MTU exchange error is caught?

Reply:
Yep, same error in both locations, initial connection is fine, but the rest of the operations will time out with this error.